spec/game_results_spec.rb

```diff
@@ -57,41 +57,45 @@
     end 
     
     describe "#draw?" do
+        before(:each) do
+            @board = Board.new 
+        end
+
         it "returns false if the board is empty" do 
-            board = Board.new
             result = GameResults.new
 
-            expect(result.draw?(board)).to eq(false)
+            expect(result.draw?(@board)).to eq(false)
         end
```

Comment:
What does your `draw` function return if the board is full and there is a winner?

Reply:
It will return true, which is not helpful. But is that relevant if I'm not checking for a draw until after I've checked if there is a winner? If it is relevant, I can add language to the draw function to  check if there is not a winner.

Reply:
I'll address in a separate PR.